main.go

```diff
@@ -35,7 +36,23 @@ func main() {
 	}
 	utils.KeployVersion = version
 	fmt.Println(logo, " ")
-	fmt.Printf("version: %v\n\n", version)
+	releaseInfo, err2 := utils.GetLatestGitHubRelease()
```

Comment:
Try to do this at root.go so that the current version print statement comes before this statement.

Reply:
If we do this at root.go, then the update message wont be visible everytime we run keploy, which was one of the original goals of this feature. Let me know if this behaviour is unintended or needs to be revised.

Reply:
Every command in keploy goes through root cmd right..?

Reply:
Refactored